src/treemodel/document.js

```diff
@@ -309,6 +319,12 @@ export default class Document {
 	 *
 	 * @event core.treeModel.Document#changesDone
 	 */
+
+	/**
+	 * Fired when a first delta is added to a batch. Can be treated as "new batch created" event.
+	 *
+	 * @event core.treeModel.Document#batch
```

Comment:
"Can be treated as new batch created event." Can be called `batchCreated` too ;)